packages/discovery/src/discovery/interactive/InteractiveDiffIgnore.ts

```diff
@@ -0,0 +1,196 @@
+import select from '@inquirer/select'
+import { ContractParameters } from '@l2beat/discovery-types'
+import chalk from 'chalk'
+
+import { DiscoveryOverridesBuilder } from '../config/DiscoveryOverridesBuilder'
+import { DiscoveryDiff } from '../output/diffDiscovery'
+
+interface Change {
+  contract: string
+  property: string
+}
+
+export class InteractiveDiffIgnore {
+  constructor(private readonly dob: DiscoveryOverridesBuilder) {}
+
+  async runForDiffs(discoveryDiffs: DiscoveryDiff[]): Promise<void> {
+    console.log(chalk.green('Starting interactive mode...'))
+
+    // Filter-out diffs without changes
+    const changesWithDiffs: Change[] = discoveryDiffs.flatMap(
+      (discoveryDiff) => {
+        const contract = discoveryDiff.name
```

Comment:
The name extraction should happen only after the diff actually exists - so after the if below.